spec/models/concerns/filterable_spec.rb

```diff
@@ -193,15 +187,6 @@
       expect(filtered.items.map(&:name)).to eq(expected_items)
     end
 
-    it "can search ignoring diacritics" do
-      all_items = BasicFilterableTestModel.where(nil)
-      expected_items = [all_items.first.name]
-
-      filtered = BasicFilterableTestModel.filter(params: ActionController::Parameters.new({ filter: { search: target_user.name.split(//).join("").tr("aeiouylszcn", "àèîôûÿłšżçñ") } }))
-
-      expect(filtered.items.map(&:name)).to eq(expected_items)
-    end
-
```

Comment:
this test had a dependency on `pg_search` and seems pretty specific to `pg_search` so I removed it